app/views/questionnaire.py

```diff
@@ -177,6 +177,7 @@ def post_block(routing_path, schema, metadata, answer_store, eq_id, form_type, c
         return _save_sign_out(routing_path, current_location, form, schema, answer_store, metadata)
 
     if form.validate():
+        _set_started_at_metadata_if_required(form, metadata)
```

Comment:
I only thought of this because another PR got reverted for breaking existing sessions - this will currently incorrectly set the `started_at` time to be the time of the first form submission after this gets deployed - it may be better to set this in the setup of new sessions, e.g. around here: https://github.com/ONSdigital/eq-survey-runner/blob/master/app/authentication/authenticator.py#L110?

Reply:
As discussed, this is not a problem!

Reply:
Ad discussed where with who?

Reply:
Between Ben and I, and it spawned the next comment in the converter about the KeyError (as that's where the actual problem was)